src/entities/crm/crm.ts

```diff
@@ -361,6 +389,14 @@ export class CRMSetupRemoteError extends BaseError {
   }
 }
 
+export class CRMCreateExternalUserFromPersonError extends BaseError {
```

Comment:
nit: can you use BaseErrorV2 with a changed signature here please. This enables forwarding API errors to the UI, which would be useful here.

Reply:
Should I then refactor all crm file with BaseErrorV2 usage?

Reply:
did it for every error class